commands/scale_cluster.go

```diff
@@ -196,8 +196,8 @@ func scaleCluster(args scaleClusterArguments) error {
 	auxParams.ActivityName = scaleClusterActivityName
 
 	_, err := ClientV2.ModifyCluster(args.clusterID, reqBody, auxParams)
-	if err != nil{
-		return  microerror.Mask(err)
+	if err != nil {
```

Comment:
Looks like somebody should get their golang editor integrations sorted. 😛

Reply:
Tell me about it, Atom is really grinding my gears the last few days.

Reply:
I use https://atom.io/packages/go-plus and have `goimports` for the formatting and stuff.

Reply:
I have exactly the same setup, I really like `go-plus` but for some reason parts of it have randomly stopped working.